src/pages/groupComparison/comparisonGroupManager/GroupCheckbox.tsx

```diff
@@ -180,7 +192,7 @@ export default class GroupCheckbox extends React.Component<IGroupCheckboxProps,
                 </span></DefaultTooltip>,
                 <DefaultTooltip overlay={"Add Currently Selected Samples"}>
                     <span
-                        onClick={this.onEditClick}
+                        onClick={this.onAddClick}
```

Comment:
since you're not adding any functionality, there's no reason not to just do `onClick={this.props.onAdd}` and above `onClick={this.props.onDelete}`